lms/djangoapps/bulk_email/tests/test_models.py

```diff
@@ -134,6 +134,21 @@ def test_render_html(self):
         context = self._get_sample_html_context()
         template.render_htmltext("My new html text.", context)
 
+    def test_render_html_xss(self):
+        template = CourseEmailTemplate.get_template()
+        context = self._get_sample_html_context()
+        context['course_title'] = "<script>alert('Course Title!');</alert>"
+        context['name'] = "<script>alert('Profile Name!');</alert>"
+        # Must have user_id and course_id present in order to do keyword substitution
+        context['user_id'] = 12345
+        context['course_id'] = "course-v1:edx+100+1"
+        message = template.render_htmltext(
+            "Dear %%USER_FULLNAME%%, thanks for enrolling in %%COURSE_DISPLAY_NAME%%.", context
+        )
+        self.assertNotIn("<script>", message)
+        self.assertIn("&lt;script&gt;alert('Course Title!');&lt;/alert&gt;", message)
+        self.assertIn("&lt;script&gt;alert('Profile Name!');&lt;/alert&gt;", message)
+
     def test_render_plain(self):
```

Comment:
Does it make sense to add a test to enforce that rendering plain text does not escape?